packages/app/src/app/components/CodeEditor/CodeMirror/elements.ts

```diff
@@ -30,7 +35,7 @@ export const CodeContainer = styled.div`
     font-family: ${props =>
       fontFamilies(props.fontFamily, 'Menlo', 'Source Code Pro', 'monospace')};
     line-height: ${props => props.lineHeight};
-    background: ${theme.background2()};
+    background: ${props => props.theme.background2()};
```

Comment:
I think we need to use `theme` here directly since this Button can be used outside of the app context.

Reply:
Ah, I see, okay, let me fix that :)